src/test/java/org/broadinstitute/hellbender/engine/FeatureInputUnitTest.java

```diff
@@ -257,4 +257,24 @@ public void testToString() {
         Assert.assertEquals(namelessGenomicsDB.toString(), "gendb://" + new File("file1").getAbsolutePath(), "String representation of nameless FeatureInput with genomicsDB path incorrect");
         Assert.assertEquals(namedGenomicsDB.toString(), "name:gendb://" + new File("file1").getAbsolutePath(), "String representation of named FeatureInput with genomicsDB path incorrect");
     }
+
+    @DataProvider(name = "IsUserSuppliedNameData")
+    public Object[][] isUserSuppliedNameData() {
+        return new Object[][] {
+                // input String, expected Feature path, expected logical name
+                {"hdfs://localhost/user/my.vcf", false},
+                {"myname:hdfs://localhost/user/my.vcf", true},
+                {"myname,key1=value1:hdfs://localhost/user/my.vcf", true},
+                {"myname//:hdfs://localhost/user/my.vcf", true}
```

Comment:
Include at least one test case with a relative path on the right side (rather than a URI)

Reply:
ok